cloudformation/application.yml

```diff
@@ -0,0 +1,405 @@
+        default: "PostgreSQL Server Port"
+      DbName:
+        default: "Database Name"
+      DbSchema:
+        default: "Database Schema"
+      DbUser:
+        default: "Authenticator Database User"
+      DbPassword:
+        default: "Authenticator Password"
+      DbAnonRole:
+        default: "Anonymous Role Name"
+      DbPool:
+        default: "Connection Pool Size"
+      MaxRows:
+        default: "Default Maximum Rows Returned"
+      PreRequest:
+        default: "Pre Request"
+      EnvironmentMode:
+        default: "Environment Mode"
+      JwtSecret:
+        default: "JWT Secret"
+    ParameterGroups:
+      - Label:
+          default: "General"
+        Parameters:
+          - ClusterName
+          - ListenerHostNamePattern
+          - EnvironmentMode
+          - JwtSecret
+          - HasHttpsListener
+      - Label:
+          default: "Production Only"
+        Parameters:
+          - DbHost
+          - DbPort
+          - DesiredCount
+      - Label:
+          default: "Container Images"
+        Parameters:
+          - PostgrestImage
+          - OpenRestyImage
+          - SandboxPostgresImage
+      - Label:
+          default: "Database"
+        Parameters:
+          - DbName
+          - DbSchema
+          - DbUser
+          - DbPassword
+          - DbAnonRole
+      - Label:
+          default: "PostgREST Settings"
+        Parameters:
+          - DbPool
+          - MaxRows
+          - PreRequest
+Conditions:
+  IsDevelopment: !Equals [ !Ref EnvironmentMode, "Development" ]
+  IsProduction: !Equals [ !Ref EnvironmentMode, "Production" ]
+  UseCustomECRRegion: !Not [ !Equals [ !Ref ECRRegion, "" ] ]
+  HasHttpsListener: !Equals [ !Ref HasHttpsListener, "Yes" ]
+Resources:
+  PgLogGroup:
+    Condition: IsDevelopment
+    Type: AWS::Logs::LogGroup
+  PgRestLogGroup:
+    Type: AWS::Logs::LogGroup
+  OpenRestyLogGroup:
+    Type: AWS::Logs::LogGroup
+  TaskDef:
+    Condition: IsDevelopment
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      ContainerDefinitions:
+        - !If
+          - IsDevelopment
+          - Name: "pg"
+            Image: !Join
+              - ""
+              -
+                - !Ref AWS::AccountId
+                - ".dkr.ecr."
+                - !If
+                  - UseCustomECRRegion
+                  - !Ref ECRRegion
+                  - !Ref AWS::Region
+                - ".amazonaws.com/"
+                - !Ref SandboxPostgresImage
+            Cpu: "50"
+            Memory: "128"
+            PortMappings:
+              - HostPort: "5432"
+                ContainerPort: "5432"
+                Protocol: "tcp"
+            Environment:
+              - Name: POSTGRES_USER
+                Value: !Ref SandboxPostgresSuperUser
+              - Name: POSTGRES_PASSWORD
+                Value: !Ref SandboxPostgresSuperUserPassword
+              - Name: POSTGRES_DB
+                Value: !Ref DbName
+              # env vars useful for our sql scripts
```

Comment:
These params apply only when running locally where the db is initialised at startup with the sql scripts. Since at startup the scripts do not run (but are executed manually by the user against the live db), these vars can be removed